RecoTracker/FinalTrackSelectors/plugins/TrackLwtnnClassifier.cc

```diff
@@ -0,0 +1,124 @@
+#include "RecoTracker/FinalTrackSelectors/interface/TrackMVAClassifier.h"
+
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+
+#include "getBestVertex.h"
+
+//from lwtnn
+#include "TrackingTools/Records/interface/TrackingComponentsRecord.h"
+#include "lwtnn/LightweightNeuralNetwork.hh"
+
+//Used for manually accepting problematic short/detached tracks to general collection
+std::map<int, float> generalCollThresholds = 
+{
+{4,0.0}, 			//initialStep
```

Comment:
values here should naturally have the same origin as the values used in the iter tracking configuration.
I don't think that a hardcoded setup can be maintained easily.

If this piece remains, it should be turned to a constexpr because non-const global statics are not allowed.

Reply:
I now changed the output from computeMVA to be pair<float,bool> and the MVA value of tracks where the DNN outputs unreliable value is set to be above the general collection threshold in the TrackMVAClassifierBase. This results in the same values from iter tracking being used.